radixdlt-core/radixdlt/src/main/java/com/radixdlt/consensus/LedgerHeader.java

```diff
@@ -139,7 +141,7 @@ private LedgerHeader(
 	) {
 		this.epoch = epoch;
```

Comment:
Maybe a check that epoch and view can't be negative? Can they be zero @LukasGasior1 @talekhinezh?

Reply:
Yes, can be zero, and that would be a good check.

Reply:
added check